packages/teleport/src/services/apps/types.ts

```diff
@@ -28,6 +28,12 @@ export interface App {
   awsConsole: boolean;
 }
 
+export interface AppsResponse {
```

Comment:
Perhaps we should choose between `type` and `interface` and stick with it. Personally I prefer `type`, because interfaces allow you to do
```typescript
interface Window {
  title: string
}

interface Window {
  ts: TypeScriptAPI
}

const src = 'const a = "Hello World"';
window.ts.transpileModule(src, {});
``` 
which is confusing and needless.

Reply:
I'd also prefer using `type` unless you REALLY need the extra features provided by interfaces

Reply:
lol, i forgot i preferred type too, i was mindlessly copying and pasting the nearest type :laughing: